llvm/lib/ProfileData/SampleProfWriter.cpp

```diff
@@ -836,7 +840,7 @@ std::error_code SampleProfileWriterBinary::writeBody(const FunctionSamples &S) {
     encodeULEB128(Sample.getSamples(), OS);
     encodeULEB128(Sample.getCallTargets().size(), OS);
     for (const auto &J : Sample.getSortedCallTargets()) {
-      StringRef Callee = J.first;
+      auto Callee = J.first;
```

Comment:
nit: auto->ProfileFuncRef for readability. See https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable